samples/python/demo.py

```diff
@@ -17,11 +17,15 @@
 from subprocess import Popen
 
 try:
-    import tkinter as tk  # Python 3
-    from tkinter.scrolledtext import ScrolledText
-except ImportError:
-    import Tkinter as tk  # Python 2
-    from ScrolledText import ScrolledText
+    if sys.version_info[0] == 3:
+        import tkinter as tk  # Python 3
+        from tkinter.scrolledtext import ScrolledText #Python3
+    elif sys.version_info[0] == 2:
+        import Tkinter as tk #Python2
+        from ScrolledText import ScrolledText #Python2
+except ImportError as importerr:
+    print("Error while importing prerequisite ",importer)
+    sys.exit()
```

Comment:
`exit()` status should not be zero (means normal exit code, no error).
Not sure about python rules here, but we can use 2 here (usually used for missed files)

Also there are several cases then window just blinks without feedback to user (missing "Press the <ENTER> key to continue...")